lib/http/apps/wallet.js

```diff
@@ -15,37 +15,50 @@ const password = process.env.PASSWORD || "coinsence";
  * @apiGroup Wallet
  * @apiVersion 1.0.0
  * 
- * @apiParam {Number} accountId Accounts unique ID.
+ * @apiParam {String[]} accountsIds Accounts unique ID.
  * 
  * @apiUse PostWalletSuccess
  * 
- * @apiUse MissingAccountIdError
- * @apiUse ExistantWalletError
+ * @apiUse MissingAccountsIdsError
  * @apiUse InternalServerError
  */
 app.post(`/`, (req, res) => {
-  createEncryptedWallet(res.locals.accountId, password).then(wallet => {
-    if (res.locals.coinsenceSigner) {
-      res.locals.coinsenceSigner.sendTransaction({
-        to: wallet.address,
-        value: ethers.utils.parseEther('0.1')
-      });
-    }
-    res.status(201).json({
-      address: wallet.address,
-      mnemonic: wallet.mnemonic
-    });
-  }).catch(e => {
-    console.log(e);
-    if(e.errno == -17) {
-      res.status(409).json({
-        error: "account already has a wallet"
+  const accountsIds = req.body.accountsIds;
+  let result = [];
+
+  if (!accountsIds) {
+    res.status(400).json({error: 'accountIds is missing'});
+  } else {
+    for(let accountId in accountsIds) {
+      createEncryptedWallet(accountId, password).then(wallet => {
+        if (res.locals.coinsenceSigner) {
+          res.locals.coinsenceSigner.sendTransaction({
+            to: wallet.address,
+            value: ethers.utils.parseEther('0.1')
+          });
+        }
+        result.push({
+          accountId: wallet.address
+          //mnemonic: wallet.mnemonic
+        });
+      }).catch(e => {
+        console.log(e);
+        if(e.errno == -17) {
+          loadWallet(accountId, password).then((wallet) => {
+            result.push({
+              accountId: wallet.address
+              //mnemonic: wallet.mnemonic
+            });
+          })
+        }
+        else {
+          res.sendStatus(500);
```

Comment:
I think we should ignore this case for now...
if one wallet fails to create we should add an empty entry.